homeassistant/components/sensor/xbox_live.py

```diff
@@ -117,5 +117,5 @@ def icon(self):
     def update(self):
         """Update state data from Xbox API."""
         presence = self._api.get_user_presence(self._xuid)
-        self._state = presence.get('state', STATE_UNKNOWN)
+        self._state = presence.get('state', None)
```

Comment:
If it's a dictionary the default returned value is `None` when the key is missing.